docs/ita.md

```diff
@@ -0,0 +1,204 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: kbs-config
+  namespace: trustee-operator-system
+data:
+  kbs-config.json: |
+    {
+        "insecure_http" : true,
+        "sockets": ["0.0.0.0:8080"],
+        "auth_public_key": "/etc/auth-secret/publicKey",
+        "attestation_token_config": {
+          "attestation_token_type": "Jwt",
+          "trusted_certs_paths": ["https://portal.trustauthority.intel.com"]
+        },
+        "repository_config": {
+          "type": "LocalFs",
+          "dir_path": "/opt/confidential-containers/kbs/repository"
+        },
+        "as_config": {
+          "work_dir": "/opt/confidential-containers/attestation-service",
+          "policy_engine": "opa",
+          "attestation_token_broker": "Simple",
+          "attestation_token_config": {
+            "duration_min": 5
+          },
+          "rvps_config": {
+            "store_type": "LocalJson",
+            "store_config": {
+              "file_path": "/opt/confidential-containers/rvps/reference-values/reference-values.json"
+            }
+          }
+        },
+        "policy_engine_config": {
+          "policy_path": "/opt/confidential-containers/opa/policy.rego"
+        },
+        "intel_trust_authority_config" : {
+          "base_url": "https://api.trustauthority.intel.com",
+          "api_key": "tBfd5kKX2x9ahbodKV1...",
+          "certs_file": "https://portal.trustauthority.intel.com"
+        }
+    }
+EOF
+```
+
+### Trustee attestation policies
+
+#### RVPS reference values
+
+```bash
+oc apply -f - << EOF
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: rvps-reference-values
+  namespace: trustee-operator-system
+data:
+  reference-values.json: |
+    [
+    ]
+EOF
+```
+
+#### Trustee resource policy
+
+```bash
+oc apply -f - << EOF
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: resource-policy
+  namespace: trustee-operator-system
+data:
+  policy.rego:
+    package policy
+    default allow = false
+    allow {
+      input["attester_type"] != "sample"
+    }
+EOF
+```
+
+### Create secrets
+
+How to create secrets to be shared with the attested clients?
+In this example we create a secret *kbsres1* with two entries. These resources (key1, key2) can be retrieved by the Trustee clients.
+You can add more secrets as per your requirements.
+
+```bash
+oc create secret generic kbsres1 --from-literal key1=res1val1 --from-literal key2=res1val2 -n trustee-operator-system
+```
+
+### Create KbsConfig CRD
+
+Finally, the CRD for the operator is created:
+
+```bash
+oc apply -f - << EOF
+apiVersion: confidentialcontainers.org/v1alpha1
+kind: KbsConfig
+metadata:
+  labels:
+    app.kubernetes.io/name: kbsconfig
+    app.kubernetes.io/instance: kbsconfig
+    app.kubernetes.io/part-of: trustee-operator
+    app.kubernetes.io/managed-by: kustomize
+    app.kubernetes.io/created-by: trustee-operator
+  name: kbsconfig
+  namespace: trustee-operator-system
+spec:
+  kbsConfigMapName: kbs-config-cm
```

Comment:
Shouldn't it be kbs-config?

Reply:
And I reflected the changes here as well, including this one that you pointed out.